OpenRA.Mods.Common/Lint/CheckPlayers.cs

```diff
@@ -59,7 +59,14 @@ public void Run(Action<string> emitError, Action<string> emitWarning, Map map)
 			foreach (var kv in map.ActorDefinitions)
 			{
 				var actorReference = new ActorReference(kv.Value.Value, kv.Value.ToDictionary());
-				var ownerName = actorReference.InitDict.Get<OwnerInit>().PlayerName;
+				var ownerInit = actorReference.InitDict.GetOrDefault<OwnerInit>();
+				if (ownerInit == null)
+				{
+					emitError("Actor {0} is not owned by any player.".F(kv.Key));
+					continue;
```

Comment:
This is a bit ugly.  You can make the check below an `else if` and avoid the continue.